BuildingSync/schema
# Feat/translation

#### Any background context you want to provide?
BuildingSync v3 is being developed with breaking changes from v2

#### What does this PR do?
Creates the translation file for transforming breaking v3 changes from v2 bsync instance documents.
This includes translations for the breaking changes that were already merged into v3:
- #335 
- #341 

#### How should this be manually tested?
See validation_spec.rb. You can run locally and see the translated v2 file in spec/output

#### What are the relevant tickets?
#347
